### PR TITLE
Add tutorial to get started with sparrowdo

### PR DESCRIPTION
**Pull request by Tyil**

### CODE REVIEW

File: _tutorials/sparrowdo-getting-started.adoc [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+=== Writing a `sparrowfile`
+As noted in the previous paragraph, for the sake of a demo I'll guide you
+through creating a `sparrowfile` to install and configure `dnsmasq` as a local
+DNS cache. Using your favourite `$EDITOR`, write the following to `sparrowfile`:
+
+[source,perl6]
+----
+package-install "dnsmasq";
+directory "/etc/dnsmasq";
+file-create "/etc/dnsmasq.conf", %(source => "$*CWD/dnsmasq.conf");
```
Comment (melezhik) [inline _tutorials/sparrowdo-getting-started.adoc]: You can even omit $*CWD if you want to, as by default sparrowdo looks for files located in the current working directory.

Comment (melezhik) [reply _tutorials/sparrowdo-getting-started.adoc]: This is true after I fix that - https://github.com/melezhik/sparrowdo/issues/25 ?

Comment (Tyil) [reply _tutorials/sparrowdo-getting-started.adoc]: I'll have to test that when I get home, but I'm not sure how awake I'll be when I get there. Possibly I can check it tomorrow. Thanks for the reading in advance.

Comment (melezhik) [reply _tutorials/sparrowdo-getting-started.adoc]: Sure, I'have applied fix for  melezhik/sparrowdo#25

Comment (spigell) [inline _tutorials/sparrowdo-getting-started.adoc]: I think you should use `content` parameter for file creation. `Source` if for coping files within one machine. If on remote host you don't have it you will get an error.
```
file-create "/etc/dnsmasq.conf", %(content => slurp "$*CWD/dnsmasq.conf");
file-create "/etc/dnsmasq.d/resolv.conf", %(content => slurp "$*CWD/resolv.conf");
```

**Comment by melezhik** [comment]
Good tutorial Patrick. Clear, concise and with plain English! Thanks again.

**Comment by Tyil** [comment]
Glad you like it :D

**Pull request merged**